ArticleTemplates/assets/js/modules/ads.js

```diff
@@ -165,6 +159,37 @@ define([
 
                 window.applyNativeFunctionCall("getMpuPosCallback");
 
+            },
+            // iOS Timer
+            iosTimer : function(time, yPos) {
+                setTimeout(function() {
+                    modules.runIosPoller(time, yPos);
+                }, 1000);
+            },
+            // iOS Count
+            runIosPoller : function(start, yPos) {
+                var thisNumber = parseInt(start, 10) + 1;
+                var yPolled;
+                if (thisNumber <= 1000) { 
```

Comment:
My own oversite as I placed a big number when initially testing, I've now removed 1000 seconds upper limit and replaced with gradual increase in time for the setTimeOut polling loop, which increments the interval time by 200mls per loop, so the poller runs less frequently the longer the user is on the article (when all assets are most likely to have downloaded even on slow connections).